ci/ray_ci/doc/test_autodoc.py

```diff
@@ -0,0 +1,49 @@
+import tempfile
+import sys
+import pytest
+
+from ci.ray_ci.doc.autodoc import Autodoc
+from ci.ray_ci.doc.api import API, AnnotationType, CodeType
+
+
+def test_walk():
+    with tempfile.TemporaryDirectory() as tmp:
+        with open(f"{tmp}/head.rst", "w") as f:
+            f.write(".. toctree:\n")
+            f.write("\n")
+            f.write("\tapi_01.rst\n")
+            f.write("\tapi_02.rst\n")
+        with open(f"{tmp}/api_01.rst", "w") as f:
+            f.write(".. autosummary::\n")
```

Comment:
nit: we can merge and just print `\n\n` instead of a dedicated `f.write` for a newline